### PR TITLE
Documentation for login page

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1257

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

File: docs/src/user/manual/login-logout/login-page.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+## Login Form
+
+![Login Form](./img/login-page-1.png)
+
+1. **Email**: Enter your registered email address in this field. This is the email associated with your account.
+2. **Password**: Input your password in this field. Passwords are case-sensitive, so ensure you enter it correctly.
+3. **Remember Me**: Check this box if you want the application to remember your login credentials for future sessions. This feature is optional and can be useful for quick and convenient access.
+4. **Login Button**: Click the "LOGIN" button to submit your credentials and access your account.
+
+## Remember Me
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:16]:
```suggestion
### Remember Me
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+The "Remember Me" checkbox allows you to opt for the application to remember your login information. When checked, you won't need to re-enter your email and password each time you visit the login page. This is particularly convenient for returning users who want a seamless login experience.
+
+## Security
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:20]:
```suggestion
### Security
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+
+The `Login Page` is designed with security in mind to protect your account and sensitive information. It uses encryption protocols and best practices to ensure that your login credentials are kept secure.
+
+
+## Logging In
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:25]:
```suggestion
### Logging In
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+The Login Page is the entry point to access your account within the application. This page provides a secure way for users to log in using their credentials. It offers a straightforward interface for entering your email address, password, and includes an optional "Remember Me" checkbox for convenience.
+
+## Login Form
+
+![Login Form](./img/login-page-1.png)
+
+1. **Email**: Enter your registered email address in this field. This is the email associated with your account.
+2. **Password**: Input your password in this field. Passwords are case-sensitive, so ensure you enter it correctly.
+3. **Remember Me**: Check this box if you want the application to remember your login credentials for future sessions. This feature is optional and can be useful for quick and convenient access.
+4. **Login Button**: Click the "LOGIN" button to submit your credentials and access your account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:14]:
```suggestion
4. **Login Button**: Click the `LOGIN` button to submit your credentials and access your account.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
...
+The "Remember Me" checkbox allows you to opt for the application to remember your login information. When checked, you won't need to re-enter your email and password each time you visit the login page. This is particularly convenient for returning users who want a seamless login experience.
+
+## Security
+
+The `Login Page` is designed with security in mind to protect your account and sensitive information. It uses encryption protocols and best practices to ensure that your login credentials are kept secure.
+
+
+## Logging In
+
+Please ensure that you enter your email and password correctly to access your account. After clicking the "LOGIN" button, you will be redirected to two factor authentication page within the application.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:27]:
```suggestion
Please ensure that you enter your email and password correctly to access your account. After clicking the `LOGIN` button, you will be redirected to the two factor authentication page within the application.
```

File: docs/src/user/manual/login-logout/login-page.md [hunk 6]
```diff
@@ -1,2 +1,30 @@
...
+
+
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login-logout/login-page.md:29]: Please add a `## Summary` title above the summary here.

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor changes requested. Good job

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 6 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Review by Jeremy-Prior — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 45 sec

**Pull request merged**